tests/test_geocoding.py

```diff
@@ -23,9 +22,8 @@ def point_of_interest():
 
 
 def test_geocoding(point_of_interest: PointOfInterest):
-    # TODO: Make this test pass by implementing GeoCodingProcessor
     subject = GeoCodingProcessor(Settings())
     result = subject.enhance([point_of_interest])
     enhanced: PointOfInterest = unsafe_perform_io(result).unwrap()[0]
-    assert enhanced.lat == '50.847608'
-    assert enhanced.lng == '16.473205'
+    assert enhanced.lat == 52.2473216
```

Comment:
You have to fake `make_geocode_request` here because you're not allowed to do actual IO in unit tests. See how it's done for Google sheets